src/engine/contactwriter.cpp

```diff
@@ -4808,6 +4875,9 @@ QContactManager::Error ContactWriter::syncUpdate(const QString &syncTarget,
             }
         }
 
+        // create incidental local constituents to store aggregated data
+        // which does not originate from the server-side contact.
+        // XXX TODO: When does this happen such that syncUpdate() is called?
```

Comment:
'aggregateAdditions' is only populated in the 'exportUpdate' variation, so it's specific to syncing changes made in the exported subset database.  I can't recall the precise situation off-hand, but it's not relevant to this change.